.buildkite/shared-pipeline-vars

```diff
@@ -6,4 +6,10 @@
 # 🎗️ If you update the image to a newer Xcode version, don't forget to also update the badge in the README.md file accordingly for consistency
 export IMAGE_ID="xcode-15.4"
 
+# Uncomment this to experiment with an Xcode beta only on PRs and TestFlight beta builds
+# (while still using the latest stable Xcode declared above for final Release Builds)
+if [[ "$PIPELINE" == "pipeline.yml" || "$BETA_RELEASE" == "true" ]]; then
```

Comment:
I just realized that this logic didn't work anyway (i.e. [this CI build associated with building this PR](https://buildkite.com/automattic/pocket-casts-ios/builds/7884#019170a4-b84b-489d-9f47-c59765467e27) still used `xcode-15.4`), I believe because the `PIPELINE` env var is not defined to any value in the case of builds triggered from commits to `trunk` or PR branches (we do rely on `pipeline.yml` being the default if `PIPELINE` env var is not defined).
So I should have written `[[ "${PIPELINE:-pipeline.yml" == "pipeline.yml" … ]]` for that test to work properly in call cases instead.

(We're going to change this whole logic anyway [based on ongoing discussions in this PR](https://github.com/Automattic/pocket-casts-ios/pull/2066#issuecomment-2299364262), but wanted to mention in case we end up still wanting to use a similar logic in our final implementation)